app/src/main/java/org/projectbuendia/client/ui/chart/PebbleExtension.java

```diff
@@ -263,7 +263,9 @@ static String formatValues(List<ObsValue> values, Format format) {
             String pattern = "" + args.get("pattern");
 
             if (input instanceof ReadableInstant) {
-                return DateTimeFormat.forPattern(pattern).print(new DateTime(input));
+                //Return time specif to time zone rather than UTC
+                return DateTimeFormat.forPattern(pattern).print(
```

Comment:
Yes, this works, and it's even nicer than what I suggested.  To make it clearer why we're doing this, though, specify the time zone like so:
```
new DateTime(input, DateTimeZone.getDefault())
```
This makes it obvious that we want to format the timestamp in the default time zone.